app/controllers/concerns/pagination_methods.rb

```diff
@@ -0,0 +1,27 @@
+module PaginationMethods
```

Comment:
I like extracting this to a concern :+1:

Reply:
merged in another pull request